### ISSUE TITLE
[feature] support collapse and expand function, loop 

### ISSUE THREAD

**Issue by dvdface** [skylot/jadx#1849]
*Describe your idea:*
When reading a long method ,  using mouse scroll up and down is a little hard to follow calling flow, could you support collapse and expand  function, function in inner class  in source code ?

**Comment by MrIkso** [comment]
@dvdface, This has been available for a long time. Update jadx to latest version.
<details>
  <summary>Preview</summary>
  
![Image](https://github.com/user-attachments/assets/0968daca-c37c-4f00-a1bf-3771983e8490)

</details>
@skylot,  close this issue
